.github/workflows/reusable-docker-test.yml

```
@@ -0,0 +1,633 @@
+              echo "The log contains the string: $LOG_STRING"
+            else
+              echo "The log does not contain the string: $LOG_STRING"
+              exit 1
+            fi
+          else
+            echo "Error: Container $CONTAINER_NAME is not running."
+            exit 2
+          fi
+
+      - name: Test liquibase wrong ENV variable
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:5:8: Quote this to prevent word splitting

(shellcheck)